### DISCUSSION TITLE
how to enable play ground for sprinng boot grahql web flux starter

### DISCUSSION THREAD

**Discussion by SavvasRameshBabu** [graphql-java/graphql-java#2828] [Q&A]
I have a project created using com.graphql-java:graphql-java-spring-boot-starter-webflux:2.0 , now  I want to enable a play ground for this.

**Comment by bbakerman** [comment]
The spring starters in graphql-java are now superseded by the official Spring support

https://spring.io/projects/spring-graphql

I think you will want to check that out
